src/types/NewsroomWebhook.ts

```diff
@@ -0,0 +1,8 @@
+export default interface NewsroomWebhook {
+    id: string;
+    name: string;
+    url: string;
+    events: string[];
```

Comment:
Let's introduce an enum for event names?

Reply:
Thanks for the suggestion, done!